.ci/scripts/functional-test.sh

```diff
@@ -25,13 +25,14 @@ STACK_VERSION=${3:-"${BASE_VERSION}"}
 BEAT_VERSION=${4:-"${BASE_VERSION}"}
 ELASTIC_AGENT_VERSION=${5:-"${BASE_VERSION}"}
 GOARCH=${GOARCH:-"amd64"}
+REPORT_PREFIX="${SUITE}_${GOARCH}_${TAGS}"
```

Comment:
```suggestion
REPORT_PREFIX=${REPORT_PREFIX:-"${SUITE}_${GOARCH}_${TAGS}"}
```

Reply:
@v1v this is what you mean? I can enclose everything in quotes too, but I'm following same approach that in the above variables